### DISCUSSION TITLE
Builtin env concating support

### DISCUSSION THREAD

**Discussion by myl7** [spencerwooo/onedrive-vercel-index#196] [Ideas]
According to #19 #195 , it does not seem rare for OneDrive to give a more than 4KB token (which is particularly unsupported by Vercel). Other than requiring users to edit code or files, how about adding builtin support to concat several envs?

The design may be:
If `REFRESH_TOKEN` ends with `<more>`, concat its value after stripping `<more>` suffix with env `REFRESH_TOKEN_1`. And so do `REFRESH_TOEKN_1`, `REFRESH_TOEKN_2`, ...

**Comment by myl7** [comment]
Ops, it seems I misunderstand the true problem: the TOTAL env size.
Sorry but anyway, #199 can mitigate it.

**Reply by spencerwooo** [reply]
yes i meant to tell you that limit applied to the total size lol
